plugins.d/Mail_Relaying/mail_relay.py

```diff
@@ -83,27 +101,51 @@ def run():
 
         while 1:
             fields = [
-                ('Host', host, field_width, field_limit),
-                ('Port', port, field_width, field_limit),
-                ('Login', login, field_width, field_limit),
-                ('Password', password, field_width, field_limit)
+                ('Host', 1, 0, host, 1, 10, field_width, field_limit),
+                ('Port', 2, 0, port, 2, 10, field_width, field_limit),
+                ('Login', 3, 0, login, 3, 10, field_width, field_limit),
+                ('Password', 4, 0, password, 4, 10, field_width, field_limit)
             ]
 
             retcode, values = console.form(TITLE, FORMNOTE, fields)
+            host, port, login, password = tuple(values)
 
             if retcode is not 'ok':
                 console.msgbox(TITLE,
                                'You have cancelled the configuration process.'
                                ' No relaying of mail will be performed.')
                 return
 
-            host, port, login, password = tuple(values)
```

Comment:
@OnGle - here's the previous line, albeit in a slightly different place.

Reply:
FWIW it was there right from the start: https://github.com/turnkeylinux/confconsole/commit/e87fda7deb2a19329578d009754af2fe93287ebe

Reply:
Oh yep fair enough, still odd to me but not by anyway :p